packages/office-ui-fabric-react/src/components/FocusZone/examples/FocusZone.Photos.Example.tsx

```diff
@@ -5,7 +5,13 @@ import { FocusZone } from 'office-ui-fabric-react/lib/FocusZone';
 import { Image } from 'office-ui-fabric-react/lib/Image';
 import './FocusZone.Photos.Example.scss';
 
-const PHOTOS = createArray(25, () => {
+interface IPhoto {
```

Comment:
Is this OK to add an interface here, or should is there an easier/better way to resolve this typedef?

Reply:
Ok with disabling for tests and examples.